### ISSUE TITLE
Not able to change name of Cluster through Plugin Config

### ISSUE THREAD

**Issue by vandanpathak-gep** [s12v/newrelic-elasticsearch#63]
Hi

While installing the service , In Config File , there was an option to give name in plugin file . We gave a different name in that but in newrelic it shows default ElasticSearch cluster name. 
Can you help me to pass on Custom name to NewRelic and should not take default name of Elastic Cluster. ?

**Comment by s12v** [comment]
could you please paste your config?

**Comment by vandanpathak-gep** [comment]
{
  "agents": [
    {
      "host" : "localhost",
      "port" : 9200,
      "username": "",
      "password": "",
      "_protocol": "Optional. Default 'http'. Rename to 'protocol' and set to 'https' if needed.",
      "_basePath": "Optional. API path on server. Default '/'. Rename to 'basePath' if you need to customize it",
      "_name": "QC_US"
    }
  ]
}

Hope U asking for Config from file Plugin.Json .

**Comment by s12v** [comment]
@vandanpathak, rename `_name` to `name`

**Comment by vandanpathak-gep** [comment]
@s12v Thanks Bro.. It Worked.. 
One Question , I have 3 Load balanced VM and all three are like node of single elastic cluster . If i install this plugin on all three VM's will there be data duplicate in NewRelic dashboard since on all three VM , Service is running ?

**Comment by s12v** [comment]
@vandanpathak, yes, the same data will be sent multiple times. I don't know how would it look like on the New Relic side. I would recommend to install it on a single node.

**Comment by vandanpathak-gep** [comment]
Ok. Why i am installing on three because if the Node in which plugin is installed goes down then data coming to newrelic will get stop so have asked.

**Comment by s12v** [comment]
@vandanpathak, yes, I understand. Well, maybe try to run it on all instances, it doesn't send so much data.

**Comment by joshbranham** [comment]
We run it on all instances in a cluster @vandanpathak
